.github/workflows/ci.yml

```
@@ -120,4 +120,20 @@ jobs:
             make || exit 1; \
             cd ..; \
           done;
-
+  cmake-windows-latest:
+    runs-on: windows-latest
+    env:
+      APACHE_LOUNGE_DISTRO_VERSION: 2.4.57
+      HTTPD_DEV_HOME: '${{ github.workspace }}\httpd-apache-lounge\Apache24'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Developer Command Prompt for Microsoft Visual C++
+        uses: ilammy/msvc-dev-cmd@v1
+      - name: Get httpd
+        run: |
+          curl.exe --output "httpd-2.4.57-win64-VS16.zip" --url "https://www.apachelounge.com/download/VS16/binaries/httpd-2.4.57-win64-VS16.zip"
+          Expand-Archive -Path "httpd-2.4.57-win64-VS16.zip" -DestinationPath "httpd-apache-lounge"
```

Perhaps like this? Not a windows expert :)

```suggestion
          curl.exe --output "httpd-%APACHE_LOUNGE_DISTRO_VERSION%-win64-VS16.zip" --url "https://www.apachelounge.com/download/VS16/binaries/httpd-%APACHE_LOUNGE_DISTRO_VERSION%-win64-VS16.zip"
          Expand-Archive -Path "httpd-%APACHE_LOUNGE_DISTRO_VERSION%-win64-VS16.zip" -DestinationPath "httpd-apache-lounge"
```

---

BTW If that has a chance of working you can press "Commit suggestion"